src/mlpack/methods/ann/layer/flexible_relu.hpp

```diff
@@ -0,0 +1,146 @@
+/**
+ * @file flexible_relu.hpp
+ * @author Aarush Gupta
+ * @author Manthan-R-Sheth
+ *
+ * Definition of FlexibleReLU layer as described by
+ * Suo Qiu, Xiangmin Xu and Bolun Cai in
+ * "FReLU: Flexible Rectified Linear Units for Improving Convolutional
+ *   Neural Networks", 2018
+ *
+ * For more information, read the following paper:
+ *
+ * @code
+ * @article{
```

Comment:
This is missing a name, also do you mind to move this into the class description block, that way, doxygen can pick this up.